tests/ok_ko.ml

```diff
@@ -0,0 +1,27 @@
+(** Compile files in "OK" and "KO".
+    REVIEW: Most tests could be removed from "lp" files, and tested directly in
+    the code. *)
```

Comment:
Should we test lambdapi features through `lp` files, or should we try to use the Ocaml API? I feel that using the ocaml api may target more precisely the feature we want to test, but it may be more cumbersome to set up.

Reply:
If it works well now as it is, no need to spend time on changing this now. Please, turn this into a "REMARK".

Reply:
As with previous remark, I removed it. Do you prefer that I leave it as a remark?